# HG changeset patch
# User Rik <rik@octave.org>
# Date 1409065542 25200
#      Tue Aug 26 08:05:42 2014 -0700
# Node ID bb0c5e182c121b25c9c3e3a655a0ab05df06c60b
# Parent  3d0f4f4ec688c6d41b3a99cc5d27b2c9f3fa3f5b
isprime.m: Update docstring to note that 1 is not prime.

isprime.m: Update docstring to note that 1 is not prime.

diff --git a/scripts/specfun/isprime.m b/scripts/specfun/isprime.m
--- a/scripts/specfun/isprime.m
+++ b/scripts/specfun/isprime.m
@@ -17,21 +17,21 @@
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn {Function File} {} isprime (@var{x})
 ## Return a logical array which is true where the elements of @var{x} are
 ## prime numbers and false where they are not.
 ##
-## A prime number is conventionally defined as a positive integer (1, 2, 3,
-## @dots{}) which is divisible only by itself and 1.  Octave extends this
-## definition to include both negative integers and complex values.  A
-## negative integer is prime if its positive counterpart is prime.  This is
-## equivalent to @code{isprime (abs (x))}.
+## A prime number is conventionally defined as a positive integer greater than
+## 1 (e.g., 2, 3, @dots{}) which is divisible only by itself and 1.  Octave
+## extends this definition to include both negative integers and complex
+## values.  A negative integer is prime if its positive counterpart is prime.
+## This is equivalent to @code{isprime (abs (x))}.
 ## 
 ## If @code{class (@var{x})} is complex, then primality is tested in the domain
 ## of Gaussian integers (@url{http://en.wikipedia.org/wiki/Gaussian_integer}).
 ## Some non-complex integers are prime in the ordinary sense, but not in the
 ## domain of Gaussian integers.  For example, @math{5 = (1+2i)*(1-2i)} shows
 ## that 5 is not prime because it has a factor other than itself and 1.
 ## Exercise caution when testing complex and real values together in the same
 ## matrix.
